components/forms/Button/Button.tsx

```diff
@@ -25,6 +25,7 @@ export const Button = (
       "gc-button--secondary": secondary,
       "gc-button--base": base,
       "gc-button--destructive": destructive,
+      "gc-button--disabled": disabled,
```

Comment:
Not sure if  this is even used with your implementation. Could it be code that you had early in the development but ended up changing strategy?

Reply:
That's exactly what happened. I'll remove it.